src/color-gamut/README.md

```diff
@@ -36,13 +36,12 @@ colorGamutElement.color = colorValue;
 
 | Variable | Type | Description |
 |----------|---------------|-------------|
-| `--gamut-color` | `<color>` | Background color of gamut indicator. Will override the color that depends on the actual gamut, so you should rarely use this directly. |
-| `--color-neutral-60` | `<color>` | Neutral background color of gamut indicator. |
```

Comment:
Gah, I didn’t mean remove this, I meant there needs to be a better variable here. I’m on it, keep an eye out so you can review.